javalib/src/main/scala/java/time/LocalDate.scala

```diff
@@ -0,0 +1,432 @@
+package java.time
+
+import scala.scalajs.js
+
+import java.time.chrono.{IsoEra, Era, IsoChronology, ChronoLocalDate}
+import java.time.temporal._
+
+final class LocalDate private (year: Int, month: Int, dayOfMonth: Int)
+    extends ChronoLocalDate with Serializable {
+  import Preconditions.requireDateTime
+  import ChronoField._
+  import ChronoUnit._
+  import LocalDate._
+
+  requireDateTime(year >= minYear && year <= maxYear,
+      s"Invalid value for year: $year")
+  requireDateTime(month > 0 && month <= 12, s"Invalid value for month: $month")
+
+  private val _month = Month.of(month)
```

Comment:
Keeping `month` and `_month` seem to be a redundant. Use only the field for `_month` and get it's value for `month`.